source/NPC.cpp

```diff
@@ -107,21 +107,31 @@ void NPC::Load(const DataNode &node)
 			stockConversation = GameData::Conversations().Get(child.Token(1));
 		else if(child.Token(0) == "ship")
 		{
-			if(child.HasChildren())
+			if(child.HasChildren() && child.Size() == 2)
 			{
+				// Loading an NPC from a save file, or an entire ship specification.
+				// The latter may result in references to non-instantiated outfits.
 				ships.push_back(make_shared<Ship>());
 				ships.back()->Load(child);
 				for(const DataNode &grand : child)
 					if(grand.Token(0) == "actions" && grand.Size() >= 2)
 						actions[ships.back().get()] = grand.Value(1);
-				if(child.Size() > 2 && ships.back()->Name().length() < 1)
-					ships.back()->SetName(child.Token(2));
 			}
 			else if(child.Size() >= 2)
 			{
+				// Loading a ship managed by GameData, i.e. "base models" and variants.
 				stockShips.push_back(GameData::Ships().Get(child.Token(1)));
 				shipNames.push_back(child.Token(1 + (child.Size() > 2)));
 			}
+			else
+			{
+				string message = "Skipping unsupported use of a ship name and and child nodes: ";
+				if(child.Size() >= 3)
+					message += "to refer to a customized ship, create a variant. and reference it here.";
+				else
+					message += "the \"ship\" token must be accompanied with the name of the base model ship.";
```

Comment:
I'm not a coding wizard but I think \"ship\" is a typo

Reply:
here's the trace in an 80x24 terminal: 
![image](https://user-images.githubusercontent.com/20871346/31576213-e1cf97f2-b0bb-11e7-898c-fdd6a8d0de12.png)

Reply:
The backslash will "escape" the doublequote, making the string print with the word ship in quotes:
![image](https://user-images.githubusercontent.com/20871346/31576266-ca948a60-b0bc-11e7-871a-78bd66de68a4.png)

Reply:
ye I understand now, + I learned something, so me thinking it was a typo is wrong